packages/evergreen-badges/src/components/Badge.js

```diff
@@ -35,14 +36,12 @@ export default class Badge extends PureComponent {
     textAlign: 'center',
     borderRadius: 2,
     isSolid: false,
-    boxSizing: 'border-box',
-    textTransform: 'uppercase',
   }
 
   render() {
     const { appearance, isSolid, ...props } = this.props
+    const appearanceStyle = BadgeAppearances[appearance]
```

Comment:
See other comment for this to make sense, this can be:

```js
const appearanceStyle = BadgeAppearances[appearance][isSolid ? 'solid' : 'default']
```

Reply:
should be the other way around.  see update `const appearanceStyle = BadgeAppearances[opacity][appearance]`